bibserver/search.py

```diff
@@ -337,8 +337,8 @@ def collection(self):
                 admin = True if metadata != None and auth.collection.update(self.current_user, metadata) else False
                 if metadata and '_display_settings' in metadata:
                     self.search_options.update(metadata['_display_settings'])
-                users = bibserver.dao.Account.query(size=1000000) # pass the userlist for autocomplete admin addition (could be ajax'd)
-                userlist = [i['_source']['_id'] for i in users['hits']['hits']]
+                users = bibserver.dao.Account.query(size=10000) # pass the userlist for autocomplete admin addition (could be ajax'd)
```

Comment:
ES throws an error (might have been a caught warning), asking specifically for that value.